commands.go

```diff
@@ -425,9 +476,14 @@ func (c *commandable) GetRange(key string, start, end int64) *StringCmd {
 	return cmd
 }
 
-func (c *commandable) GetSet(key, value string) *StringCmd {
-	cmd := NewStringCmd("GETSET", key, value)
-	c.Process(cmd)
+func (c *commandable) GetSet(key string, value interface{}) *StringCmd {
```

Comment:
http://www.darkcoding.net/software/go-the-price-of-interface/ :) Assuming you are doing 300k GETs per second: 120ns * 300,000 = 3.6ms. Not the end of the world, but again, my goal for this client is to be 'as fast as possible', but it may not apply to everyone else.